Description: Recognize const value "lcdfilterlegacy",
 used in Ubuntu before upstream introduced "lcdlegacy"
---
 src/fcname.c |    1 +
 1 file changed, 1 insertion(+)

--- a/src/fcname.c
+++ b/src/fcname.c
@@ -198,6 +198,7 @@ static const FcConstant _FcBaseConstants
     { (FcChar8 *) "lcddefault",	    "lcdfilter",    FC_LCD_DEFAULT },
     { (FcChar8 *) "lcdlight",	    "lcdfilter",    FC_LCD_LIGHT },
     { (FcChar8 *) "lcdlegacy",	    "lcdfilter",    FC_LCD_LEGACY },
+    { (FcChar8 *) "lcdfilterlegacy",	    "lcdfilter",    FC_LCD_LEGACY },
 };
 
 #define NUM_FC_CONSTANTS   (sizeof _FcBaseConstants/sizeof _FcBaseConstants[0])
